### PR TITLE
Fix icons in all components

### PR DESCRIPTION
**Pull request by LaChope**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,7 @@
     "dist/"
   ],
   "dependencies": {
-    "intelligent-tree-select": "~0.8.3"
-  },
-  "peerDependencies": {
+    "intelligent-tree-select": "~0.8.3",
```
Comment (blcham) [inline package.json]: What is the reason to move from peerDependencies to dependencies ?

**Review by blcham — CHANGES_REQUESTED** [review]
See my comment.

File: package.json (hunk 1 above)
Comment (LaChope) [reply package.json]: As smart-components needs s-forms to run and its dependencies I thought it makes more sense to put it like this.

Here is where my thoughts come from:

> Dependencies
> 
> Dependencies are listed in the package.json file in a dependencies object.
> 
> When you add a package in dependencies, you are saying:
> 
> My code needs this package to run.
> If this package doesn’t already exist in my node_modules directory, then add it automatically.
> Furthermore, add the packages that are listed in the package’s dependencies. These packages are called transitive dependencies.
> 
> 
> Peer Dependencies
> 
> Peer Dependencies are listed in the package.json file in a peerDependencies object.
> 
> By adding a package in peerDependencies you are saying:
> 
> My code is compatible with this version of the package.
> If this package already exists in node_modules, do nothing.
> If this package doesn’t already exist in the node_modules directory or it is the wrong version, don’t add it. But, show a warning to the user that it wasn’t found.

Comment (blcham) [reply package.json]: https://stackoverflow.com/questions/26737819/why-use-peer-dependencies-in-npm-for-plugins

> When should you use peer dependencies?
> - When you are building a library to be used by other projects, and
> - This library is using some other library, and
> - You expect/need the user to work with that other library as well


Is this reasonable answer to u ? definitely question it !

Comment (blcham) [reply package.json]: @LaChope  resolving this (we had discussion)

**Pull request merged**